cg/cli/clean.py

```diff
@@ -267,7 +267,7 @@ def remove_invalid_flow_cell_directories(context: CGConfig, failed_only: bool, d
     status_db: Store = context.status_db
     demux_api: DemultiplexingAPI = context.demultiplex_api
     housekeeper_api: HousekeeperAPI = context.housekeeper_api
-    trailblazer_api: TrailblazerAPI = context.trailblazer_api
+    trailblazer_api: TrailblazerClient = context.trailblazer_api
```

Comment:
```suggestion
    trailblazer_client: TrailblazerClient = context.trailblazer_client
```

Reply:
Yes 👍  Lets do it in a separate PR, this one is only intended to rename the class

Reply:
https://github.com/Clinical-Genomics/cg/pull/2291